docs/advanced/advanced_examples.rst

```diff
@@ -40,4 +40,16 @@ Publication style:
 
 .. image:: ../img/hep_examples_dataMC_stacked_publication.svg
    :alt: Publication style
-   :width: 500
\ No newline at end of file
+   :width: 500
+
+
+Color palette
+=============
+
+When displaying quantities such as yields or intensities, it is recommended to utilize Perceptually Uniform Sequential Colormaps like the cubehelix colormap for accurate representation.
+
+We provide the ``cubehelix_palette()`` (see `here <https://plothist.readthedocs.io/en/latest/documentation/documentation.html#plothist.plotters.cubehelix_palette>`_) function to create such colormap. With ``ncolors = 11``:
+
```

Comment:
Maybe write the code to produce the colormap plot, including the `from plothist import cubehelix_palette` to help the user.